tests/io/test_noaagml.py

```diff
@@ -128,3 +128,25 @@ def test_read_surfrad():
     assert ds['temperature'].values[0] == 2.0
     assert 'standard_name' in ds['temperature'].attrs
     assert ds['temperature'].attrs['standard_name'] == 'air_temperature'
+
+
+def test_read_gml_aerosol():
+    for datatype in ['AEROSOL', 'aerosol', None]:
+        ds = read_gml(sample_files.EXAMPLE_GML_AEROSOL_NAS, datatype='AEROSOL')
```

Comment:
Does datatype need to be plugged in somewhere? Looks like this is just a repeating loop

Reply:
@kenkehoe does datatype='AEROSOL' need to be datatype=datatype for the loop?